### DISCUSSION TITLE
Fluent v9 React Dialog Width

### DISCUSSION THREAD

**Discussion by KC-Anisha** [microsoft/fluentui#29588] [Q&A]
Hi all,

Does anyone know how one can set the min-width and max-width (and heights) for a [Dialog](https://react.fluentui.dev/?path=/docs/components-dialog--default&globals=storybook_fluentui-react-addon_theme:web-light)

I think this was an option in v8 but not sure in v9.

Thanks!

**Comment by makarandkeer** [comment]
I am also looking for increasing width of the Dialog. Anyone found the way to do it?

**Reply by KC-Anisha** [reply]
I was able to get it working but in a hacky way by adding this to my CSS

`.fui-DialogSurface {
    min-width: 80% !important;
}`

**Reply by makarandkeer** [reply]
I have found one more way !! 
set `style` on `DialogSurface`

` <DialogSurface  style={{ "min-width": dialogWidth }}`

**Reply by khmakoto** [reply]
Yes, in general, you need to set the style on the `DialogSurface`. We would recommend creating a class name with `makeStyles` and adding it via `className` instead of `style` for perfomance purposes though 😄.

**Reply by makarandkeer** [reply]
@khmakoto In my case I have to change `min-width` dynamically hence I am using `style`. Is there way to use `makeStyles` but still change the styles dynamically?

**Reply by khmakoto** [reply]
Ahhh, you would need to use a css variable in the `makeStyles` call but still set it via `style` if you really wanted to use `makeStyles`, but I agree that it is almost the same thing at that point.
